RelationalAI/Client.cs

```diff
@@ -469,7 +472,7 @@ public async Task<TransactionAsyncResult> DeleteModelsAsync(string database, str
             // slow-path
             var transactionResponse = await Policy
                 .HandleResult<TransactionAsyncSingleResponse>(r => !r.Transaction.State.IsFinalState())
-                .RetryForeverWithBoundedDelay()
+                .RetryForeverWithBoundedDelay(startTime, 0.2)
```

Comment:
defaults to 0.1 in the rest of the code, but then here it uses 0.2.  Just curious here.. obviously this should be a config option but that can wait for a later time since we need this quickly.  I would just conform whatever we're trying to conserve.   Also, 20% or even 10% is a lot of overhead.  I thought our goal was 1-5%? I can see that for short transactions that would make the delay super small, but maybe its better to just parameterize with a minimum delay in addition (say 500ms from the CommonPolicies code), and between the minimum and maximum, obey the threshold and set the threshold lower

Reply:
I think not providing a default value is a better option, I was leaning toward that
The 20% overhead is inspired from the go-sdk, for the julia sdk we use 10% of the total transaction runtime as a delay